Makefile

```diff
@@ -69,11 +69,11 @@ back:
 ifeq ($(INSTALL_DEV_DEPENDENCIES), TRUE)
 	@echo "INSTALL_DEV_DEPENDENCIES=$(INSTALL_DEV_DEPENDENCIES)"
 	@echo "Installing composer dependencies, including dev ones"
-	$(call php, composer install --prefer-dist -o)
+	$(call php, composer install -o)
```

Comment:
see [comment](https://github.com/skilld-labs/skilld-docker-container/pull/267/files#r449446084) below

Reply:
`--prefer-dist` is actually unnecessary cause the config in `composer.json` takes priority so it doesn't really matter if we include or remove this parameter, to keep this PR atomic I think it is ok to leave unchanged